### PR TITLE
Update README.md

### LINKED ISSUE #1021: Typo in README
I encountered this issue in the Readme. md file of this project, It's in this screenshot,
Screenshot 2023-05-29 at 4 39 05 PM

But according to me, It should be:- "I want to perform data analysis and data science-related tasks in JavaScript and Node.js, similar to how I might use Python, Julia, R, and MATLAB."

### PR DESCRIPTION
**Pull request by nithinkatta**
Resolves #1021 .

## Description

> What is the purpose of this pull request?

This pull request:

-   fixes a typo in the main project README.

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #1021

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ While this project's installation instructions defaults to using [npm][npm] for
 
 ### User Stories
 
--   I want to perform **data analysis** and/or **data science** related tasks in JavaScript and Node.js, similar to how I might use IPython, Julia, R, and/or MATLAB.
+-   I want to perform **data analysis** and **data science**-related tasks in JavaScript and Node.js, similar to how I might use Python, Julia, R, and MATLAB.
```
Comment (kgryte) [inline README.md]:
```suggestion
-   I want to perform **data analysis** and **data science** tasks in JavaScript and Node.js, similar to how I might use Python, Julia, R, and MATLAB.
```

**Review by kgryte — APPROVED** [review]

**Comment by kgryte** [comment]
@nithinkatta Thanks for submitting this PR. In order to merge, we need you to check the box, indicating that you have read the contributing guidelines and agreed to have this change incorporated into stdlib under its associated licenses.

Once you agree, then we can merge this PR.

**Comment by nithinkatta** [comment]
Yes you can merge the PR. Will it consider in ssoc , as I'm participant in it

**Comment by Planeshifter** [comment]
Merged in!  

I don't know whether this PR will count towards SSOC, you will have to check in with the organizers of that event.

**Comment by nithinkatta** [comment]
Okay, Thank you @kgryte @Planeshifter

**Pull request merged**